Kickstarter-iOS/Views/Controllers/ProjectNavBarViewController.swift

```diff
@@ -122,21 +122,15 @@ public final class ProjectNavBarViewController: UIViewController {
 
     self.watchProjectViewModel.outputs.generateImpactFeedback
       .observeForUI()
-      .observeValues { [weak self] in
-        self?.saveButton.generateImpactFeedback(style: .light)
```

Comment:
Team notice that this is a thing of past (these proxies were just duplicating things and made a lot of reasoning about our code harder than need be).

So previously the flow would be
```swift
button.generateImpactFeedback
  -> UIFeedbackGenerator.ksr_impact()
    -> UIImpactFeedbackGenerator(style: style).impactOccurred()`
```

now we're removing one layer and doing
```swift
UIFeedbackGenerator.ksr_generateImpactFeedback()
  -> AppEnvironment.current.impactFeedbackGenerator.impactOccured
```
(please not that the naming is now more consistent) & (this also allows us to test things)

Cc: @justinswart @cdolm92 @ifbarrera @Scollaco

Reply:
After some feedback from Justin it's now even simpler...we only have one publicly available function for each haptic type.

So what was previously this

```swift
button.generateImpactFeedback
  -> UIFeedbackGenerator.ksr_impact()
    -> UIImpactFeedbackGenerator(style: style).impactOccurred()
```

is now
```swift
generateImpactFeedback()
```